docs/jetstream/jetstream.md

```diff
@@ -57,7 +57,7 @@ for which all clients have complete data.
 
 When analyzing experiments, the following steps are executed for each experiment:
 
-![Experiment analyis steps](https://github.com/mozilla/experimenter-docs/blob/main/static/img/jetstream/analysis-steps.png)
+<img src="/img/jetstream/analysis-steps.png" alt="Experiment analyis steps" width="350"/>
```

Comment:
Good catch! :+1:

It's a pretty weak/naive dual-grep, but it looks like this might be the last/only remaining image with a fully qualified domain:

```sh
git grep "\.png" | grep -E "https?://"

docs/jetstream/jetstream.md:![Experiment analyis steps](https://github.com/mozilla/experimenter-docs/blob/main/static/img/jetstream/analysis-steps.png)
```